Assets/MRTK/SDK/Features/UX/Scripts/Collections/GridObjectCollectionSizeFitter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    [ExecuteInEditMode]
+    public class GridObjectCollectionSizeFitter : MonoBehaviour
+    {
+        [SerializeField]
```

Comment:
Can you move this script to MRTK/Examples/Experimental/GridObjectCollectionFitter/Scripts/